onnxruntime/core/providers/cpu/tensor/upsample.h

```diff
@@ -69,6 +69,12 @@ class UpsampleBase {
                                                      ? info.GetAttrOrDefault<std::string>("coordinate_transformation_mode", "half_pixel")
                                                      : "asymmetric";
     coordinate_transform_mode_ = StringToCoordinateTransformationMode(coordinate_transform_mode_name);
+    if (opset >= 13) {
+      // TODO: Do we actually need to enforce this ? Might seem overly pedantic.
+      ORT_ENFORCE(coordinate_transform_mode_name != "tf_half_pixel_for_nn",
+                  "`tf_half_pixel_for_nn` is deprectaed since opset 13");
+    }
```

Comment:
nit: deprectaed -> deprecated. 

I would think a warning is OK vs. failing to run the model.

Reply:
Fixed the typo.

Used the default logger to log as there doesn't seem to be a way to access the session logger via OpKernelInfo (there seems to be an interface via OpKernelContext)